lib/matplotlib/contour.py

```diff
@@ -1532,7 +1542,7 @@ def _check_xyz(self, args, kwargs):
         Exception class (here and elsewhere).
         """
         x, y = args[:2]
-        self.ax._process_unit_info(xdata=x, ydata=y, kwargs=kwargs)
+        kwargs = self.ax._process_unit_info(xdata=x, ydata=y, kwargs=kwargs)
```

Comment:
In this case, it's passed as a mutable value, so you don't technically need to return it. It should be the same object coming out as going in.